### PR TITLE
New command: Check

### PR DESCRIPTION
**Pull request by AndrewJWin**
Serves to check the guild to ensure Misaki has key permissions for proper functionality.

### CODE REVIEW

File: commands/system/check.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    super(client, {
+      name: "check",
+      description: "Check if Misaki has the required permissions.",
+      usage: "check",
+      category: "check"
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+    message.channel.send(`${message.guild.me.permissions.has("EMBED_LINKS") ? ":white_check_mark: ║ I have the ability to Embed Links." : ":negative_squared_cross_mark: ║ I do **not** have the ability to Embed Links."}\n\n${message.guild.me.permissions.has("MANAGE_MESSAGES") ? ":white_check_mark: ║ I have the ability to Manage Messages." : ":negative_squared_cross_mark: ║ I do **not** have the ability to Manage Messages."}\n\n${message.guild.me.permissions.has("ADD_REACTIONS") ? ":white_check_mark: ║ I have the ability to Add Reactions." : ":negative_squared_cross_mark: ║ I do **not** have the ability to Add Reactions."}\n\n${!message.guild.me.permissions.has(["EMBED_LINKS","MANAGE_MESSAGES","ADD_REACTIONS"]) ? "Misaki is missing a few permissions, some commands may not work properly if these permissions are missing." : "Misaki has all the permissions that are currently required for flawless operation!"}`);}
```
Comment (YorkAARGH) [inline commands/system/check.js:13]: Able to break this up into multiple lines (for ease of reading)

**Comment by MrJacz** [comment]
This might be the time to propose per command permissions,
`botPerms` an array of Permissions for example `SEND_MESSAGES`, have an inhibitor (like klasa, no I'm not saying to use I'm just referring to it.) basically per command you have the defined **required** permissions then in message event check that the bot has all perms, this could also use the Permission class for PermissionResolvable (e.g Array, Permission, Bitfield) but this being said, this command is quite bloated being one message...

File: commands/system/check.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const Command = require(`${process.cwd()}/base/Command.js`);
+class Ready extends Command {
```
Comment (MrJacz) [inline commands/system/check.js:2]: this isnt Ready???

File: commands/system/check.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  constructor(client) {
+    super(client, {
+      name: "check",
+      description: "Check if Misaki has the required permissions.",
+      usage: "check",
+      category: "check"
```
Comment (MrJacz) [inline commands/system/check.js:8]: this would be `system` not `check`

File: commands/system/check.js (hunk 1 above)
Comment (MrJacz) [inline commands/system/check.js:13]: Break this up........

File: commands/system/check.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+    message.channel.send(`${message.guild.me.permissions.has("EMBED_LINKS") ? ":white_check_mark: ║ I have the ability to Embed Links." : ":negative_squared_cross_mark: ║ I do **not** have the ability to Embed Links."}\n\n${message.guild.me.permissions.has("MANAGE_MESSAGES") ? ":white_check_mark: ║ I have the ability to Manage Messages." : ":negative_squared_cross_mark: ║ I do **not** have the ability to Manage Messages."}\n\n${message.guild.me.permissions.has("ADD_REACTIONS") ? ":white_check_mark: ║ I have the ability to Add Reactions." : ":negative_squared_cross_mark: ║ I do **not** have the ability to Add Reactions."}\n\n${!message.guild.me.permissions.has(["EMBED_LINKS","MANAGE_MESSAGES","ADD_REACTIONS"]) ? "Misaki is missing a few permissions, some commands may not work properly if these permissions are missing." : "Misaki has all the permissions that are currently required for flawless operation!"}`);}
+}
+
+module.exports = Ready;
```
Comment (MrJacz) [inline commands/system/check.js:16]: This isnt ready...

**Review by MrJacz — CHANGES_REQUESTED** [review]

**Comment by AndrewJWin** [comment]
Pointless due to #70

**Pull request closed without merging**